cloudflare/resource_cloudflare_access_keys_configuration.go

```diff
@@ -0,0 +1,94 @@
+package cloudflare
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	cloudflare "github.com/cloudflare/cloudflare-go"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func resourceCloudflareAccessKeysConfiguration() *schema.Resource {
+	return &schema.Resource{
+		Read:   resourceCloudflareAccessKeysConfigurationRead,
+		Create: resourceCloudflareAccessKeysConfigurationCreate,
+		Update: resourceCloudflareAccessKeysConfigurationUpdate,
+		Delete: resourceCloudflareKeysConfigDelete,
+		Importer: &schema.ResourceImporter{
+			State: resourceCloudflareAccessMutualTLSCertificateImport,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"key_rotation_interval_days": {
+				Type:     schema.TypeInt,
+				Optional: true,
+				Computed: true,
```

Comment:
`Computed` and `Optional` are mutually exclusive and you can't combine them. Need to decide on which one better suits the attribute here.

Reply:
Sure you dont mean `Computed` and `Required`? 

https://github.com/cloudflare/terraform-provider-cloudflare/blob/master/cloudflare/resource_cloudflare_access_application.go#L30

Reply:
you're right, i had remembered point 2 at https://www.terraform.io/docs/extend/schemas/schema-behaviors.html#optional in reverse